src/library/ebuild-backend.c

```diff
@@ -0,0 +1,335 @@
+#ifndef _DEFAULT_SOURCE
+#define _DEFAULT_SOURCE
```

Comment:
This is going to need changing - specifying both `_DEFAULT_SOURCE` and `_POSIX_C_SOURCE` isn't quite right.

Reply:
Yeah, definitely dev shrapnel. I'm sure that it can be checked and set globally during configure. Advice welcome before I start searching.

Reply:
autoconf can do `AC_USE_SYSTEM_EXTENSIONS` which approximately does "the best the system can offer" in terms of GNU extensions and such